### ISSUE TITLE
lanceError (arrow) - caused by typing issue in input dataframe

### ISSUE THREAD

**Issue by mrubash1** [lancedb/lancedb#64]
lancedb error on version 0.1.2

error below is solvable by forcing dtypes of
```
def force_types_of_df(df):
    desired_dtypes = {
        'id': 'string',
        'comment_text': 'string',
        'toxic':'int64',
        'severe_toxic':'int64',              
        'obscene':'int64',                   
        'threat' :'int64',                   
        'insult' :'int64',                   
        'identity_hate' :'int64',            
        'dataset':'string',
        'comment_text_processed':'string',
        'jailbreak':'int64'
        }
    # Apply the desired data types, forcing the conversion and inserting NaN for values that can't be converted
    for column, dtype in desired_dtypes.items():
        if dtype in ['int64']:
            df[column] = pd.to_numeric(df[column], errors='coerce', downcast='integer')
        elif dtype == 'datetime64':
            df[column] = pd.to_datetime(df[column], errors='coerce')
        else:
            df[column] = df[column].astype(dtype)
    return df
``` 

![lance-error](https://user-images.githubusercontent.com/4754702/236928331-1e18d893-85b6-451a-9c62-fc330607d20a.png)

**Comment by mrubash1** [comment]
Also needed to clean nulls
```
df.replace('', np.nan, inplace=True)
df = df.dropna()
```

**Comment by changhiskhan** [comment]
addressed in #846
